datafusion/expr/src/aggregate_function.rs

```diff
@@ -411,3 +411,23 @@ impl AggregateFunction {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use strum::IntoEnumIterator;
+
+    #[test]
+    // Test for AggregateFuncion's Display and from_str() implementations.
+    // For each variant in AggregateFuncion, it converts the variant to a string
+    // and then back to a variant. The test asserts that the original variant and
+    // the reconstructed variant are the same.
+    fn test_display_and_from_str() {
```

Comment:
This test is mostly copied from the test for `BuiltinScalarFunction`.
https://github.com/apache/arrow-datafusion/blob/f3c9009e50fdb5811c522bfb07ba29ac04cd1d22/datafusion/expr/src/built_in_function.rs#L1625